images/win/scripts/Installers/Validate-Git.ps1

```diff
@@ -3,18 +3,21 @@
 ##  Desc:  Validate Git for Windows
 ################################################################################
 
-if((Get-Command -Name 'git') -and (Get-Command -Name 'bash') -and (Get-Command -Name 'awk') -and (Get-Command -Name 'git-lfs'))
+if((Get-Command -Name 'git') -and (Get-Command -Name 'bash') -and (Get-Command -Name 'awk') -and (Get-Command -Name 'git-lfs') -and (Get-Command -Name 'hub'))
 {
     Write-Host "$(git version) on path"
     Write-Host "$(git-lfs version) on path"
+
+    $strHubVersion = $(hub version)[1]
+    Write-Host "${strHubVersion} on path"
+    $hubVersion = ($strHubVersion -split " ")[2]
```

Comment:
I think we should keep only Write-Host here: `Write-Host "$(hub version | Select-String "hub version") on path"`

Retrieving version should be done below to be consistent with other tools:
```
if( $(hub version) -match  'hub version (?<version>.*)' )
{
   $hubVersion = $Matches.version
}
```

Reply:
Done:
https://github.com/andy-mishechkin/virtual-environments/commit/684f2a4e0134e768185991a7a9ef0fe9f6c91bbd